tests/pass/metavalues.ncl

```diff
@@ -1,12 +1,7 @@
 let {check, Assert, ..} = import "testlib.ncl" in
 
 [
-  (10 | default | Num) == 10,
-
-  # Check the correct handling of the update of thunks containing enriched values (see issue
-  # https://github.com/tweag/nickel/issues/123)
-  let x = {a = (fun x => (1 | default)) 1} in
-    builtin.seq (x.a) ((x & {a=2}).a) == 2,
```

Comment:
Same as in the other test: the removed test doesn't make sense for RFC005, which precludes the propagation of metadata.